garage/tf/algos/batch_polopt.py

```diff
@@ -63,6 +74,106 @@ def log_diagnostics(self, paths):
         self.policy.log_diagnostics(paths)
         self.baseline.log_diagnostics(paths)
 
+    def process_samples(self, itr, paths):
```

Comment:
please audit this function to make sure all of this data is still used by the algorithm. some of it might not be. some might be used only for logging (keep those)

Reply:
Currently `advantages` and `env_infos` are not in use. Can I delete them?

Reply:
env_infos no, advantages yes